frontend-react/src/utils/Analytics.ts

```diff
@@ -2,6 +2,7 @@ import { getAppInsights } from "../TelemetryService";
 
 export enum EventName {
     TABLE_FILTER = "Table Filter",
+    SESSION = "Session",
```

Comment:
VERY minor point, but should this be `SESSION_DURATION` + `"Session Duration"`?  We may want to add other session-related events in the future.